src/lib/components/Toggle.svelte

```diff
@@ -11,14 +11,38 @@
   const dispatch = createEventDispatcher();
 
   const id = nextElementId("toggle-");
+
+  let input: HTMLInputElement | undefined;
+
+  const onKeyDown = ({ code }: KeyboardEvent) => {
+    if (code !== "Space") {
+      return;
+    }
+
+    input?.click();
```

Comment:
> Is it because the checkbox has display: none?

Yes.

Reply:
Not sure it is a better solution. If we do so, assuming it would work out, then we would need extra CSS and maybe even a state too to highlight correctly focus, hover and other feedback of the related UI components. One or the other...

Reply:
That said, here I just realized `disabled` is not handled and should been added to handle the keyboard event in such case.

Let me know if you rather like going CSS etc. and we close the PR (and comeback to this when we've got time) or otherwise I also add a `if` to handle disable state.

I'm fine either way, your call.

Reply:
Either way is fine with me.